stable/yugaware/templates/service.yaml

```diff
@@ -40,199 +40,28 @@ spec:
   {{- if and (eq .Values.yugaware.service.type "LoadBalancer") (.Values.yugaware.service.ip) }}
   loadBalancerIP: "{{ .Values.yugaware.service.ip }}"
   {{- end }}
-{{- end }}
+{{- if .Values.yugaware.service.postgres }}
 ---
-apiVersion: apps/v1
-kind: StatefulSet
+apiVersion: v1
+kind: Service
 metadata:
-  name: {{ .Release.Name }}-yugaware
+  name: {{ .Release.Name }}-postgres
+  {{- if .Values.yugaware.service.annotations }}
+  annotations:
+    {{ toYaml .Values.yugaware.service.annotations | indent 4 }}
+  {{- end }}
   labels:
     app: {{ .Release.Name }}-yugaware
     chart: {{ template "yugaware.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Values.helm2Legacy | ternary "Tiller" (.Release.Service | quote) }}
 spec:
-  serviceName: {{ .Release.Name }}-yugaware
-  replicas: {{ .Values.yugaware.replicas }}
+  ports:
+  - name: postgres
+    port: 5432
+    targetPort: 5432
   selector:
-    matchLabels:
-      app: {{ .Release.Name }}-yugaware
-  template:
-    metadata:
-{{- if .Values.yugaware.pod.annotations }}
-      annotations:
-{{ toYaml .Values.yugaware.pod.annotations | indent 8 }}
+    app: {{ .Release.Name }}-yugaware
+  type: ClusterIP
```

Comment:
let's make the service type and annotations configurable just like the other service

Reply:
I wanted to keep this patch small as possible.
IMO adding same features as for YW makes it harder to maintain and remove in future.
But ok, let's to make it a bit smarter.